src/iad.cs

```diff
@@ -172,7 +172,8 @@ interface ADInterstitialAdDelegate {
 		void WillLoad (ADInterstitialAd interstitialAd);
 	}
 
-	[Category, BaseType (typeof (MPMoviePlayerController))]
+	[Category (ignoreStaticWarnings: true)]
```

Comment:
@rolfbjarne @spouliot Do we really want this to be obsoleted/error out? There is one case where our users can't avoid this. 

If a third party binding tries to extend a X.I.dll type there is no workaround, they need to use the (null as Foo) because they won't be able to inline it.